set.mm

```diff
@@ -732651,6 +732663,410 @@ The modulo (remainder) operation - extension
+
+    ${
+      $d S x z $.  $d X x $.  $d Y x $.
+      $( An element of the preimage of a function value is an element of the
+         domain of the function with the same value as another element of the
+         premimage.  (Contributed by AV, 9-Mar-2024.) $)
+      elsetpreimafvbi $p |- ( ( F Fn A /\ S e. P /\ X e. S )
+                     -> ( Y e. S <-> ( Y e. A /\ ( F ` Y ) = ( F ` X ) ) ) ) $=
+        ( wfn wcel cfv wceq wa wb ccnv cv wi fniniseg eleq2 csn cima wrex eqeq2
+        anbi2d eqcoms sylan9bb ex adantld sylbid bibi1d rexlimivw elsetpreimafv
+        imbi12d syl5ibr syl11 3imp ) FCJZEDKZGEKZHEKZHCKZHFLZGFLZMZNZOZEFPAQFLZ
+        UAUBZMZACUCURUTVGRZUSVJURVKRACURVKVJGVIKZHVIKZVFOZRURVLGCKZVDVHMZNVNCVH
+        GFSURVPVNVOURVPVNURVMVBVCVHMZNZVPVFCVHHFSVRVFOVHVDVHVDMVQVEVBVHVDVCUDUE
+        UFUGUHUIUJVJUTVLVGVNEVIGTVJVAVMVFEVIHTUKUNUOULABCDEFIUMUPUQ $.
+
+      $( The elements of the preimage of a function value have the same
+         function values.  (Contributed by AV, 5-Mar-2024.) $)
+      elsetpreimafveqfv $p |- ( ( F Fn A /\ ( S e. P /\ X e. S /\ Y e. S ) )
+                                -> ( F ` X ) = ( F ` Y ) ) $=
+        ( wfn wcel cfv wceq wi w3a wa elsetpreimafvbi simpr eqcomd syl6bi 3imp2
+        3exp ) FCJZEDKZGEKZHEKZGFLZHFLZMZUCUDUEUFUINUCUDUEOUFHCKZUHUGMZPZUIABCD
+        EFGHIQULUHUGUJUKRSTUBUA $.
+
+      $( If an element of the domain of the function has the same function
+         value as an element of the preimage of a function value, then it is an
+         element of the same premimage.  (Contributed by AV, 9-Mar-2024.) $)
+      eqfvelsetpreimafv $p |- ( ( F Fn A /\ S e. P /\ X e. S )
+                      -> ( ( Y e. A /\ ( F ` Y ) = ( F ` X ) ) -> Y e. S ) ) $=
+        ( wfn wcel w3a cfv wceq wa elsetpreimafvbi biimprd ) FCJEDKGEKLHEKHCKHF
+        MGFMNOABCDEFGHIPQ $.
+    $}
+
+    ${
+      $d A y $.  $d F y $.  $d P y $.  $d S x z $.  $d S y $.  $d X x y $.
+      $( An element of the preimage of a function value expressed as a
+         restricted class abstractionn.  (Contributed by AV, 9-Mar-2024.) $)
+      elsetpreimafvrab $p |- ( ( F Fn A /\ S e. P /\ X e. S )
+                                -> S = { x e. A | ( F ` x ) = ( F ` X ) } ) $=
+        ( vy wfn wcel w3a cv cfv wceq crab wa elsetpreimafvbi fveqeq2 elrab
+        syl6bbr eqrdv ) FCJEDKGEKLZIEAMZFNGFNZOZACPZUCIMZEKUHCKUHFNUEOZQUHUGKAB
+        CDEFGUHHRUFUIAUHCUDUHUEFSTUAUB $.
+
+      $d P x y $.
+      $( The image of an element of the preimage of a function value is the
+         singleton constisting of the function value at one of its elements.
+         (Contributed by AV, 5-Mar-2024.) $)
+      imaelsetpreimafv $p |- ( ( F Fn A /\ S e. P /\ X e. S )
+                                -> ( F " S ) = { ( F ` X ) } ) $=
+        ( vy wcel w3a cv cfv csn cima wceq wrex wa 3adant3 3ad2ant1 fveq2 sneqd
+        wfn ccnv fvelsetpreimafv imaeq2d eqeq2d cbvrexv sylibr crn cin 3ad2ant3
```

Comment:
Same here, use [cbvrexvw](https://us.metamath.org/mpeuni/cbvrexvw.html) instead of [cbvrexv](https://us.metamath.org/mpeuni/cbvrexvw.html) to avoid ax-13.
<pre>
        ( vy wcel w3a cv cfv csn cima wceq wrex wa 3adant3 3ad2ant1 fveq2 sneqd
        wfn ccnv fvelsetpreimafv imaeq2d eqeq2d cbvrexvw sylibr imaeq2 3ad2ant3
        crn cin fnfun funimacnv syl elsetpreimafvbi wi wss fnfvelrn snssd df-ss
        wfun sylib simp3 eqtrd 3expib sylbid imp 3eqtrd rexlimdv3a mpd ) FCUCZE
        DJZGEJZKZEFUDZILZFMZNZOZPZIEQZFEOZGFMZNZPZVMVNWCVOVMVNREVQALZFMZNZOZPZA
        EQWCABCDEFHUEWBWLIAEVRWHPZWAWKEWMVTWJVQWMVSWIVRWHFUAUBUFUGUHUISVPWBWGIE
        VPVREJZWBKWDFWAOZVTFULZUMZWFWBVPWDWOPWNEWAFUJUKVPWNWOWQPZWBVMVNWRVOVMFV
        CWRCFUNVTFUOUPTTVPWNWQWFPZWBVPWNWSVPWNVRCJZVSWEPZRZWSABCDEFGVRHUQVMVNXB
        WSURVOVMWTXAWSVMWTXAKZWQVTWFVMWTWQVTPZXAVMWTRZVTWPUSXDXEVSWPCVRFUTVAVTW
        PVBVDSXCVSWEVMWTXAVEUBVFVGTVHVISVJVKVL $.
</pre>

Reply:
Done